pkg/api/repository.go

```diff
@@ -27,22 +27,33 @@ type RepositoryBulkCreateResponse struct {
 	Repository *RepositoryResponse `json:"repository"` // Repository object information
 }
 
+func (r *RepositoryRequest) FillNonNilValues() {
+	defaultVersions := []string{"any"}
+	defaultArch := "any"
+	if r.DistributionVersions != nil && len(*r.DistributionVersions) == 0 {
+		r.DistributionVersions = &defaultVersions
+	}
+	if r.DistributionArch != nil && *r.DistributionArch == "" {
```

Comment:
I don't think arch is needed, right? That field tags works for gorm. Version was the only problematic one.

Reply:
In which, this function could be `FillDefaultVersions()` or something like that.

Reply:
Without this I was seeing the arch on creation being saved as "". 
I don't those tags do anything 😐

Reply:
Gotcha. It turns out this default field tag only works on Create in gorm.